tests/ci_tests/code_formatter/crewai/test_crewai_agents.py

```diff
@@ -0,0 +1,310 @@
+"""Tests for coffee_maker/code_formatter/crewai/agents.py"""
+
+from unittest import mock
+
+import pytest
+from crewai import Agent
+
+from coffee_maker.code_formatter.crewai import agents
+from coffee_maker.code_formatter.crewai.tools import PostSuggestionToolLangAI
+
+
+class TestLLMConfiguration:
+    """Tests for LLM configuration"""
+
+    def test_llm_exists(self):
+        """Test that llm object is initialized"""
+        assert agents.llm is not None
+
+    def test_llm_initialization_error(self, monkeypatch):
+        """Test that missing API keys raise a runtime error."""
+        monkeypatch.delenv("GEMINI_API_KEY", raising=False)
+        monkeypatch.delenv("GOOGLE_API_KEY", raising=False)
+        monkeypatch.delenv("COFFEE_MAKER_GEMINI_API_KEY", raising=False)
```

Comment:
Added a docstring to the method, describing its purpose.
```suggestion
    def test_returns_dict_with_senior_engineer(self):
```